velox/connectors/hive/HiveDataSink.h

```diff
@@ -42,6 +42,8 @@ class LocationHandle : public ISerializable {
     kNew,
     /// Write to an existing table.
     kExisting,
+    /// Write to a temporary table.
+    kTemporary,
```

Comment:
I feel this will not be needed once we support bucketed unpartitioned table write (just reusing `kExisting`).  

@xiaoxmeng Do we have a plan for supporting bucketed unpartitioned table write?

Reply:
@Yuhta , @xiaoxmeng : I debugged more cases for CTE and there is more work required for Append here. I'm moving this to draft mode again. Apologize for this.